sample.txt

```diff
@@ -1,5 +1,6 @@
-This is sample text.
+his is sample text.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/litmon/danger-sample/blob/f3c0167a0ced43ef13484ae60bcf1d9d3896ab9b/sample.txt#L1"></span>It is typo</td>
    </tr>
  </tbody>
</table>